### PR TITLE
Add readme file - issue #79

### PR DESCRIPTION
**Pull request by mosheeven**
Add README file for vagrant - session #2, challenge #1

### CODE REVIEW

File: vagrant/2/1/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Use:
```
Comment (lmilbaum) [inline vagrant/2/1/README.md]: I would suggest you use the README file in vagrant/1/1 as a template. Just copy it and add the status and ssh commands.

File: vagrant/2/1/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+ - "vagrant status" to see the running machines.
+ - "vagrant ssh" to ssh the machine. 
+
+In case there are more then one machine use:
```
Comment (lmilbaum) [inline vagrant/2/1/README.md]: This section is not needed for this challenge. The Vagrantfile has only one machine configured.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: vagrant/2/1/README.md (hunk 1 above)
Comment (mosheeven) [reply vagrant/2/1/README.md]: Thanks for the feedback.

**Comment by mosheeven** [comment]
Hi @lioramilbaum,
I updated the README file per the convention.

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**